cost/azure/databricks/rightsize_compute/azure_databricks_rightsize_compute.pt

```diff
@@ -261,6 +264,7 @@ end
 
```

Comment:
Line 227 - I know this is not specific to this change, but:

 "datasource code blocks found with out of order fields.
Fields should be in the following order: auth, pagination, verb, scheme, host, path, query, header, body, body_field, ignore_status"